fabric/forgero-fabric-compat/src/main/resources/assets/forgero/lang/en_us.json

```diff
@@ -1,21 +1,71 @@
+  "forgero.config.group.recipes_upgrades_and_loot.title": "Recipes, upgrades, and loot",
+  "forgero.config.group.recipes_upgrades_and_loot.title.tooltip": "Configuration for recipes, upgrades, and loot.",
+  "forgero.config.disableVanillaRecipes": "Disable vanilla recipes",
+  "forgero.config.disableVanillaRecipes.tooltip": "Disables Minecraft's vanilla tool recipes.",
+  "forgero.config.enableCustomRecipeDeletion": "Enable custom recipe deletion",
+  "forgero.config.enableCustomRecipeDeletion.tooltip": "Enables deletion of recipes via custom IDs, using a datapack system.",
+  "forgero.config.enableRecipesForAllSchematics": "Enable recipes for all schematics",
+  "forgero.config.enableRecipesForAllSchematics.tooltip": "Enables recipes for all schematics, making all schematics craftable.",
+  "forgero.config.enableUpgradeInCraftingTable": "Enable upgrade in crafting table",
+  "forgero.config.enableUpgradeInCraftingTable.tooltip": "Enables upgrading tools in a crafting table, for example, from wood to metal.",
+  "forgero.config.disableVanillaLoot": "Disable vanilla loot",
+  "forgero.config.disableVanillaLoot.tooltip": "Stops vanilla tools from appearing in vanilla/modded loot pools.",
+  "forgero.config.disableVanillaTools": "Disable vanilla tools",
+  "forgero.config.disableVanillaTools.tooltip": "Disables crafting recipes of vanilla tools.",
+  "forgero.config.convertVanillaRecipesToForgeroTools": "Convert vanilla recipes to Forgero tools",
+  "forgero.config.convertVanillaRecipesToForgeroTools.tooltip": "Repurposes vanilla tool recipes to create Forgero tools instead of vanilla tools.",
+  "forgero.config.convertVanillaToolLoot": "Convert vanilla tool loot",
+  "forgero.config.convertVanillaToolLoot.tooltip": "Swaps vanilla tools with Forgero tools when they appear in vanilla/modded loot pools.",
+  "forgero.config.group.repairing.title": "Repairing",
+  "forgero.config.group.repairing.title.tooltip": "Configuration for tool repairing.",
+  "forgero.config.enableUnbreakableTools": "Enable unbreakable tools",
+  "forgero.config.enableUnbreakableTools.tooltip": "Makes tools never break when having 0 durability, instead giving them the broken status and allowing the tools to be repaired.",
+  "forgero.config.enableRepairKits": "Enable repair kits",
+  "forgero.config.enableRepairKits.tooltip": "Enables repair kits, used for storing materials in a pouch, for quick and easy repair of your tools on the go.",
+  "forgero.config.group.attributes.title": "Attributes",
+  "forgero.config.group.attributes.title.tooltip": "Configuration for tool attributes.",
+  "forgero.config.baseSoulLevelRequirement": "Base soul level requirement",
+  "forgero.config.baseSoulLevelRequirement.tooltip": "The base level needed to apply souls to tools.",
+  "forgero.config.useEntityAttributes": "Use entity attributes",
+  "forgero.config.useEntityAttributes.tooltip": "Enables entity attributes for tools, which have special functionality.",
```

Comment:
This setting toggles if misc attributes like Mining speed, Mining Level, Durability should be displayed like entity attributes, or inside forgero's "attribute" section